tutorials/video_classification_example/finetune.py

```diff
@@ -0,0 +1,255 @@
+from pathlib import Path
+from argparse import Namespace
+from torchvision.transforms._transforms_video import CenterCropVideo
+from pytorchvideo.data import LabeledVideoDataset
+from pytorchvideo.data.clip_sampling import UniformClipSampler
+import pytorch_lightning as pl
+import torch
+from pytorchvideo.models.head import create_res_basic_head
+from torch import nn
+from torch.optim import Adam
+
+# HACK
+from train import *
```

Comment:
Assuming you'll fix this! Feel free to move the kinetics data module out of the other tutorial so that both can use it.

Reply:
yes! already fixed :)